libs/db-browser/src/main/java/com/oceanbase/tools/dbbrowser/schema/DBSchemaAccessor.java

```diff
@@ -81,6 +81,20 @@ default List<String> showTables(String schemaName) {
      */
     List<DBObjectIdentity> listTables(String schemaName, String tableNameLike);
 
+    /**
+     * Show all external table names list in the specified schema
+     */
+    default List<String> showExternalTables(String schemaName) {
+        return showExternalTablesLike(schemaName, null);
+    }
+
+    List<String> showExternalTablesLike(String schemaName, String tableNameLike);
+
+    /**
+     * List all external table as BObjectIdentity in the specified schema
+     */
+    List<DBObjectIdentity> ListExternalTables(String schemaName, String tableNameLike);
```

Comment:
how about index, constraints, partitions and etc.?

Reply:
Reuse the table implementation. For index, constraints, partitions and etc, there is no requirement that it must be a basic table